### ISSUE TITLE
Format inventory screen

### ISSUE THREAD

**Issue by tagniam** [tagniam/Turn#22]
Right now the inventory looks like this. 

```
*----------- INVENTORY -----------*
Level 2                 0/100 xp
| Arrows: [10]
| Potions: [0]
| Bombs: [3]
| Whetstones: [0]
| Weapon strength: [100%]
| Wealth: [48] coins
*---------------------------------*
```
3 things need to be implemented:
1) Put a divider between the level/xp line and the items
2) Draw a right side to the inventory box
3) xp needs to be aligned to the right, with 1 space padding

So it would end up looking like this diagram, labelled:

```
*----------- INVENTORY -----------*
| Level 2                0/100 xp | <- 3
+---------------------------------+ <- 1
| Arrows: [10]                    |
| Potions: [0]                    |
| Bombs: [3]                      | <- 2
| Whetstones: [0]                 |
| Weapon strength: [100%]         | 
| Wealth: [48] coins              |
*---------------------------------*
```

You will have to do some character counting when implementing 2 and 3, since as the values add digits, less spaces will be needed.

**Comment by SoumyadeepJana** [comment]
Working on it!

**Comment by sgrG24** [comment]
can i also work on it??

**Comment by SoumyadeepJana** [comment]
I don't think chracter counting is needed if you keep a mximum width

**Comment by ldaw** [comment]
I made a functional solution for this issue, I'll submit a PR soon.

**Comment by ldaw** [comment]
Just submitted my PR. Let me know if there are any issues, I didn't get to test it with the full project but it worked perfectly with a driver I wrote.

**Comment by tagniam** [comment]
Fixed by #31
